e2e/nextjs/.env

```diff
@@ -1,3 +1,3 @@
-NEXT_PUBLIC_HIGHLIGHT_PROJECT_ID='1jdkoe52'
-NEXT_PUBLIC_HIGHLIGHT_OTLP_ENDPOINT='http://localhost:4318'
-NEXT_PUBLIC_HIGHLIGHT_BACKEND_URL='https://localhost:8082/public'
+NEXT_PUBLIC_HIGHLIGHT_PROJECT_ID='1383'
+# NEXT_PUBLIC_HIGHLIGHT_OTLP_ENDPOINT='http://localhost:4318'
+# NEXT_PUBLIC_HIGHLIGHT_BACKEND_URL='https://localhost:8082/public'
```

Comment:
this is probably what's failing the e2e test - we expect all the e2e tests to route to the local deploy